modules/uncertainty/R/plots.R

```diff
@@ -117,11 +117,11 @@ plot_sensitivity <- function(sa.sample, sa.spline, trait, y.range = c(0, 50), me
   post.x <- seq(from = min(sa.sample), to = max(sa.sample), length.out = LENGTH_OUT)
   
   saplot <- saplot + ## plot spline function
-    ggplot2::geom_line(ggplot2::aes(x, y), data = data.frame(x = post.x, y = sa.spline(post.x)), size = linesize) + 
+    ggplot2::geom_line(ggplot2::aes(x=post.x, y=sa.spline(post.x)), data = data.frame(x = post.x, y = sa.spline(post.x)), size = linesize) + 
```

Comment:
This isn't doing quite what it looks like -- this makes ggplot go find `post.x` in the enclosing environment and ignore its `data` argument, which has the potential to cause surprises later. I believe you should be able to use `x = .data$x, y = .data$y` here same as in dplyr functions.

Reply:
understood! I'll make the necessary changes.

Reply:
resolved